packages/@orbit/jsonapi/src/utils/handle-operations-response.ts

```diff
@@ -0,0 +1,47 @@
+import { TransformBuilder } from '@orbit/data';
+// import { pushPayload, localIdFromRecordIdentity } from 'react-orbitjs';
+
+import { JSONAPIOperationsPayload } from '../lib/operations/types';
+import { JSONAPIDocument } from '..';
+
+export function handleOperationsResponse(store: Store, data: JSONAPIOperationsPayload | JSONAPIDocument) {
```

Comment:
renamed from: https://github.com/sillsdev/appbuilder-portal/pull/504/files#diff-b1db239ff87fc25a6c5bf67802c3383eR8

usage here: https://github.com/sillsdev/appbuilder-portal/pull/504/files#diff-c34425931f52700f34e1f36189ca2ebfR34

this takes any JSONAPI Document and shoves into the in-memory store.

Not sure how to adapt this for the way data flows between sources in orbit. :-\